codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/middleware/AWSSigningMiddleware.kt

```diff
@@ -57,12 +65,22 @@ open class AWSSigningMiddleware(
     }
 
     private fun middlewareParamsString(op: OperationShape): String {
```

Comment:
This is now the single place that parameters to create a Swift `SigV4Config` are generated, based on params that are passed into this middleware.